packages/@okta/vuepress-site/data/oauth2-error-codes.json

```diff
@@ -519,13 +519,27 @@
     "internalErrorCode" : "client_assertion_invalid_secret_length",
     "errorCode" : "invalid_client",
     "errorDescription" : "The client secret is too short to verify a JWT HMAC."
+  }, {
+    "title" : "Client assertion jwks not configured",
+    "statusCode" : 401,
+    "statusReasonPhrase" : "Unauthorized",
+    "internalErrorCode" : "client_assertion_jwks_not_configured",
+    "errorCode" : "invalid_client",
+    "errorDescription" : "The client does not have a JWKSet configured, but the client_assertion requires one."
```

Comment:
does not > doesn't

Reply:
FYI, if we make this update, we need to inform dev to make this change on their side, too. @vanngo-okta @christinechoy-okta

Reply:
@brianduffield-okta is right about this. @vanngo-okta, maybe ignore my comments on contractions when it's in dev-created error codes.

Reply:
Not updating error descriptions at this time.